03-Bash/fix_owner/fix_owner.sh

```diff
@@ -1,4 +1,27 @@
-#!/bin/bash
+#!/bin/bash 
+
+declare -i force
+force=0
+
+function printHelp()
+{
+    echo "$0 - check user\`s"
+    echo "-f, --force - fix without asknowledgement,"
+    echo "-h, --help - print these options."
+    exit 0
+}
+
+# check options
+if [ $# -gt 0 ]; then
+    case $1 in 
+        -h) printHelp;; 
```

Comment:
You can use operator '|' for case alternatives.

Reply:
This request was not satisfied.

Reply:
Used '|'